web/templates/confirm_dialog/confirm_resend_invite.hbs

```diff
@@ -4,3 +4,6 @@
         {{#*inline "z-email"}}<strong>{{email}}</strong>{{/inline}}
     {{/tr}}
 </p>
+<p>
+    This will not change the expiration time for this invitation.
```

Comment:
This needs to be tagged for translation, like every other string in the app.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html for details.